pyrealm/pmodel/pmodel.py

```diff
@@ -215,11 +217,18 @@ def __init__(
         # Set context specific defaults for kphio to match Stocker paper
         if kphio is None:
             if not self.do_ftemp_kphio:
-                self.init_kphio = 0.049977
+                self.init_kphio = np.array(0.049977)
```

Comment:
Could these values be pulled out into a constant rather than using magic numbers?

Reply:
I've got some inbound code that changes the `kphio` part completely, so I _think_ these will disappear in the near future.